### ISSUE TITLE
Feature Request: Saving options for resulting images:

### ISSUE THREAD

**Issue by tandefelt** [divamgupta/diffusionbee-stable-diffusion-ui#80]
It would be a nice option to have an option to 1) save the resulting images shown by the interface, straight to desktop/assigned folder, 2) possibly with some intelligent naming convention. This could enable the end user not having to click via tons of files, and getting carpal tunnel from writing names and versioning numbers etc :)..   I assume this would mean a core name, and a rolling series of numbers for max 14 files, if one works with larger batches.. (similar to some video/image edit tools) etc.. :),

**Comment by jemueller** [comment]
agree, the new version is amazing with being able to generate multiple images at the same time, but its painful to have to click each one and name/save it individually... 

an advanced saving feature would be ideal.... but even just the ability to have all the images saved to the desktop with one click at once would be a huge improvement

thank you again for making this app

**Comment by dngrlux** [comment]
In Finder, hit command + shift + G, then go to /tmp/samples. From there, you can drag the samples folder into your sidebar for quick access. You'll need to copy your favorites somewhere else, as /tmp is temporary and will eventually be wiped.

<img width="471" alt="image" src="https://user-images.githubusercontent.com/12442027/191594393-4544a345-fabd-4c05-bd0a-5ef4619f6bb3.png">

**Comment by dngrlux** [comment]
Looks like with version 0.2.0 they are now loosely saved in the /tmp folder, and no longer in a subfolder. Bug?

**Comment by jemueller** [comment]
ah yes, ok they're in the base /tmp folder now... thanks for this info, i can use it to batch copy save all the files after generation instead of having to click each button and dialog box

**Comment by tandefelt** [comment]
Hi, dngrlux, major thanks!!.. I wondered already whether there was an autosave, so.. taking my request back.. unless, there is a way to add a free user chosen download folder etc.. maybe still simpler, even on desktop etc.

**Comment by jemueller** [comment]
yeah, just having the ability to choose where the files get dropped would be good enough, like i could point it at the desktop and then organize/rename them on my own time later for the good ones i want to keep

**Comment by LePips** [comment]
I think this and all saving requests can be aggregated under https://github.com/divamgupta/diffusionbee-stable-diffusion-ui/issues/80.
